frontend/src/store/modules/students.js

```diff
@@ -0,0 +1,36 @@
+// initial state
+// shape: [{ id, quantity }]
+const state = () => ({
+  students: [],
+  //checkoutStatus: null
+})
+
+// getters
+const getters = {
+
+}
+
+// actions
+const actions = {
+    // loadStudents({commit}) {
+    //     fetch("http://localhost:3001/students/").then(result => {
+    //     commit('SAVE_STUDENTS', result.data);
+    //   }).catch(error => {
+    //         return;  
+    //   });
+    // },
+
+}
+
+// mutations
+const mutations = {
+
+}
+
+export default {
+  namespaced: true,
```

Comment:
i recommend not to `modules` maybe we can remove this first?

Reply:
Removed!